package.json

```diff
@@ -21,9 +20,10 @@
     "setup": "npm install && npm run flow-typed",
     "format": "prettier --write --ignore-unknown .",
     "format:check": "prettier --check .",
-    "test": "npm run format:check && npm run lint && npm run flow-typed && npm run flow && npm run jest && npm run karma",
+    "test": "npm run format:check && npm run lint && npm run flow-typed && npm run flow && npm run test:unit",
     "webpack": "babel-node --plugins=transform-es2015-modules-commonjs ./node_modules/.bin/webpack --progress",
-    "jest": "jest test/server --env=node --no-cache --collectCoverageFrom='server/' --coverageDirectory='coverage/jest' --coverage --verbose --runInBand --silent=false",
+    "test:unit:watch": "vitest",
+    "test:unit": "vitest run --coverage",
```

Comment:
Including `run` to `vitest` means it will exit once finished. We want that for CI runs because the standard `vitest` will keep it in `watch` mode and keep the thread open, which isn't good for CI.